osmosis-installer.py

```diff
@@ -6,11 +6,24 @@
 import random
 import argparse
 import sys
-from sys import argv
-from os import remove
+from enum import Enum, auto
 
 # self-destruct file after first call
-remove(argv[0])
+os.remove(sys.argv[0])
+
+class NetworkVersion(str, Enum):
+    MAINNET = "v11.0.1"
+    TESTNET = "v11.0.1"
+
+class NetworkType(str, Enum):
+    MAINNET = "1"
+    TESTNET = "2"
+    LOCALOSMOSIS = "3"
+
+class NodeType(str, Enum):
+    FULL = "1"
+    CLIENT = "2"
+    LOCALOSMOSIS = "3"

```

Comment:
I added this enum to improve readability.
Let me know if you like them so we can extend them to other options.

I think they make the code a little easier to understand because as they better express the user choices.

```python
# It's more clear ...
if nodeName and networkType == NetworkType.MAINNET and node == NodeType.FULL:
```

```python
# than this.
if nodeName and networkAns == "1" and node == "1":
```

Reply:
I 1000% love this, TYSM!!!

Reply:
<3 great. I'll add them to the rest!